types/drift-zoom/index.d.ts

```diff
@@ -0,0 +1,61 @@
+// Type definitions for drift-zoom 1.4
+// Project: https://github.com/imgix/drift
+// Definitions by: Piotr Błażejewicz <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/drift-zoom/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me